.github/workflows/unit-tests-workflow.yml

```diff
@@ -39,10 +39,17 @@ jobs:
         with:
           path: kibana/plugins/anomaly-detection-kibana-plugin
       - name: Bootstrap the plugin
+        continue-on-error: true
+        run: |
+          cd kibana/plugins/anomaly-detection-kibana-plugin
+          yarn kbn bootstrap
+      - name: Bootstrap the plugin again
```

Comment:
Any reason to bootstrap the plugin twice?

Reply:
This is the issue I've been trying to figure out for several days. Could be a bug with Yarn as well. Problem is identical on my local and in github runner. Basically, `yarn kbn bootstrap` is tries to reference the created `node_modules` directory before created. After it runs once, it is created and bootstrap can install all dependencies correctly.